Yoast/javascript
# Adapt specs after adding exception stem

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adapts Czech specs to reflect changes in morphology data.

## Relevant technical choices:

* Accompanying premium configuration PR: https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/229]
* The changes in the gold standard list relating to forms of `kočka` have to do with previously made changes in the functionality.
 
## Test instructions

This PR can be tested by following these steps:

* All specs should pass and there should be 0 errors when checking the gold standard list.
* In the content analysis app, add a text of at least 150 words, set `děti` as your keyphrase, and use the following forms in the text: `děti, dítě, dítěte, dětem, dítěti, dítětem, dětech`. All of these forms should be recognized.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
